chall01/dmukaliy.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/python3
+import sys
+
+def error():
+	sys.stdout.write("usage: ./dmukaliy.py <a-zA-Z string>\n")
+	exit()
+
+def morse(string):
+	alphabet = {" ":" ", "a":".-", "b":"-...", "c":"-.-.", "d":"-..", "e":".", "f":"..-.", "g":"--.", "h":"....",
+		"i":"..", "j":".---", "k":"-.-", "l":".-..", "m":"--", "n":"-.", "o":"---", "p":".--.", "q":"--.-",
+		"r":".-.", "s":"...", "t":"-", "u":"..-", "v":"...-", "w":".--", "x":"-..-", "y":"-.--", "z":"--.."}
+	for char in string:
+		if char not in alphabet.keys():
+			error()
+	for char in string:
+		sys.stdout.write(alphabet[char])
+	sys.stdout.write("\n")
```

Comment:
you should avoid printing char by char but append alphabet[char] to a string and then print the string